docs/[Codepoints]-Test-case-design-of-Codepoints.adoc

```diff
@@ -0,0 +1,641 @@
+= terasoluna-gfw-codepoints/CodePoints
+
+== CodePointsを使った文字種チェックテスト
+
+テストケースクラス: `org.terasoluna.gfw.functionaltest.app.codepoints.CodePointsTest`
```

Comment:
- [ ] Some package names are inlined and some are not, so it's a good idea to keep them consistent.

Reply:
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/1105/files?diff=unified&w=0#diff-eecfec5433160712dbb7ddebd9be303534bc8de04cf82fb20a220b24c7db8a8cR7
> テストケースクラス:\`org.terasoluna.gfw.functionaltest.app.date.DateTest\`

There are some places, like here, where they tried to make it inline but it is not inline.

Reply:
256bf16f389d7deba7ec481647afc8fa5bb47196
Fixed inlines.